src/modules/search/components/SearchForm.tsx

```diff
@@ -37,6 +37,39 @@ export interface SearchFormProps {
 
 const defaultSearchKeys = ['textSearch', 'projects'];
 
+export const validateSearchFormInput = (values: Record<string, any>) => {
+  const requiredValueSet = omitBy(
+    pick(values, [
+      'personalId',
+      'firstName',
+      'lastName',
+      'ssnSerial',
+      'dob',
+      'textSearch',
+    ]),
+    (e) => isNil(e) || isEmpty(e)
+  );
+
+  const errors: Record<keyof typeof requiredValueSet, string> = {};
+
+  if (isEmpty(requiredValueSet)) {
+    errors.textSearch =
+      'Please enter a search term or specific search criteria';
+  } else {
+    if (
+      values.textSearch &&
+      typeof values.textSearch === 'string' &&
+      isEmpty(omit(requiredValueSet, ['textSearch'])) &&
+      values.textSearch.length < 4
+    ) {
+      errors.textSearch =
+        'Please enter more than three characters or add more specific search criteria';
```

Comment:
The button below says "Advanced Search" so lets keep the language consistent

```suggestion
        'Please enter more than three characters or add advanced search criteria';
```

Reply:
Fixed